gl_context/src/common.rs

```diff
@@ -75,6 +76,27 @@ impl GLContextBuilder {
         self
     }
 
+    /// Sets the major version.
+    /// This is presently only relevant on Windows.
+    pub fn version_major(&mut self, version: u8) -> &mut Self {
+        self.gl_attributes.version_major = version;
+        self
+    }
+
+    /// Sets the minor version.
+    /// This is presently only relevant on Windows.
+    pub fn version_minor(&mut self, version: u8) -> &mut Self {
+        self.gl_attributes.version_minor = version;
+        self
+    }
+
+    /// Sets if the window should use the sRGB color space.
```

Comment:
This should probably say "Sets if the `context` should use the sRGB color space" instead of "window".

Most people will use one window, but I think technically it's possible to swap the context to another window if the window shares the same pixel buffer settings as the context.

Reply:
Fixed!